### ISSUE TITLE
Truncate file names longer than 100 in length

### ISSUE THREAD

**Issue by vsoch** [NeuroVault/NeuroVault#202]
The maximum characters that are allowed in a filename is 100. In the case that a user uploads a folder of images with lengths > allowed, none of the images are uploaded and there is no feedback as to why. It would be good to give feedback so the user can appropriately change the title lengths!

**Comment by vsoch** [comment]
Feedback is given when upload is done via "edit image" (single) page.

**Comment by vsoch** [comment]
![longtitle](https://cloud.githubusercontent.com/assets/814322/7216748/600107bc-e5be-11e4-992a-2740b21e7893.png)
title is trying to escape from the page

**Comment by vsoch** [comment]
Issue is also on edit_images page:
![issue](https://cloud.githubusercontent.com/assets/814322/7216846/119413b8-e5c2-11e4-98c2-321cc14ea9e4.png)

Here is fix for both:
![fix](https://cloud.githubusercontent.com/assets/814322/7216847/16a8e0e0-e5c2-11e4-8ff5-6a18ff29b9de.png)

New additions are "word-break" to deal with the overflow, and "line-height" so it's not crowded. The span3 (the div above) can also be set with overflow: "hidden" if you just want to truncate, but I think the first option is best.

**Comment by chrisfilo** [comment]
Could anyone replicate this in a dev environment?

**Comment by vsoch** [comment]
Yes, I can confirm that the interface hangs. Is there somewhere I can find the actual error? I don't see anything in nginx or uwsgi logs.

**Comment by chrisfilo** [comment]
That is weird. We can just truncate the names before adding them in
upload_folder.
